site/_data/eleventyComputed.js

```diff
@@ -305,13 +305,25 @@ module.exports = {
         .sort(propSort({ prop: 'data.title', lowercase: true }))
         .sort(propSort({ prop: 'data.weight', fallback: 0 }))
         .map((i) => ({
+          eyebrow: get(i, 'data.tags[0]'),
```

Comment:
```suggestion
          eyebrow: get(i, 'data.tags[1]'),
```

Because we use [data directories](https://www.11ty.dev/docs/data-template-dir/) as part of our 11ty data cascade to insert `case study` into each of our case studies, the we should use the second item, which is going to be the first descriptive tag of the case study itself